storage-units/src/indexfile.rs

```diff
@@ -50,16 +50,28 @@ fn offset_hashes(bloom_size: u32) -> u64 {
 }
 
 // calculate the file offset from where the offsets are stored
-fn offset_offsets(bloom_size: u32, number_hashes: u32) -> u64 {
-    offset_hashes(bloom_size) + HASH_SIZE as u64 * number_hashes as u64
+fn offset_offsets(bloom_size: u32, number_hashes: u32, offset_type: &IndexOffsetType) -> u64 {
+    let standard = offset_hashes(bloom_size) + HASH_SIZE as u64 * number_hashes as u64;
+    match offset_type {
+        IndexOffsetType::Standard => standard,
+        IndexOffsetType::Unsorted => standard + OFF_SIZE as u64 * number_hashes as u64,
+    }
 }
 
 pub type IndexOffset = u32;
 
+#[derive(Clone, Debug)]
+pub enum IndexOffsetType {
+    Standard,
+    Unsorted,
+}
```

Comment:
Discussed with @vincenthz - adding stuff to index files is undesirable at this moment, cuz of expected changes. Will change this whole logic a bit to store the unsorted offsets in this file: https://github.com/input-output-hk/rust-cardano/blob/master/storage/src/config.rs#L72 

This means that reading logic for unsorted offsets (un-indexed) will be very different from sorted offsets (indexed), and there will be no need for logic-switches like this one